src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipOptionsType.php

```diff
@@ -49,15 +49,31 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TranslatableType::class,
                 [
                     'type' => TextType::class,
+                    'label' => $this->trans('Delivery prefix', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans('Prefix used for delivery slips.', 'Admin.Orderscustomers.Help'),
```

Comment:
The more I look at this tooltip, the more I think we should get rid of it. It doesn't add any additional information to the label.